scripts/cibuild

```diff
@@ -25,19 +23,19 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
     if [ "${1:-}" = "--help" ]; then
         usage
     else
-        ./scripts/test
+        docker-compose up -d database
 
-        pushd "${DIR}"
+        ./scripts/test
 
         echo "Building JAR for API subproject"
-        ./sbt api/assembly
-
-        popd
+        docker-compose \
```

Comment:
Running via Compose on Jenkins to utilize OpenJDK 11. Shouldn't be much overhead.